cinema/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255, null=True)

```

Comment:
Using `null=True` on `CharField` is not recommended because it introduces two possible values for 'no data': `NULL` and an empty string. It's better to use `blank=True` if you want to allow empty values.